cmd/root.go

```diff
@@ -34,28 +34,32 @@ to quickly create a Cobra application.`,
 			os.Exit(1)
 		}
 
-		root, err := dox.Publish("", "", repoRoot, dryRun) // generated root page
+		_, err = dox.PrePublish("", "", dryRun) // generate blank root page
 		if err != nil {
 			fmt.Fprintf(os.Stderr, "error: %s\n", err)
 			os.Exit(1)
 		}
```

Comment:
I don't think a `PrePublish` step is necessary for the root page.

Reply:
We still need it in order to create the root page. Running dox on a "fresh" repo with no published pages would fail because it will try to update the root page which doesn't exist.

Reply:
Gotcha.